tests/functional/legacy_publishing/test_litezip.py

```diff
@@ -65,3 +65,47 @@ def test_publish_litezip(
         .bindparams(moduleid=collection.id, major_version=2, minor_version=1))
     inserted_tree = db_engines['common'].execute(stmt).fetchone()[0]
     compare_legacy_tree_similarity(inserted_tree['contents'], tree)
+
+
+def test_publish_litezip_collectionxml_only(
+        content_util, persist_util, app, db_engines, db_tables):
+    # Insert initial collection and modules.
+    collection, tree, modules = content_util.gen_collection()
+    modules = list([persist_util.insert_module(m) for m in modules])
+    collection, tree, modules = content_util.rebuild_collection(collection,
+                                                                tree)
+    collection = persist_util.insert_collection(collection)
+
+    # ... remove second element from the tree ...
+    tree.pop(1)
+
+    collection, tree, modules = content_util.rebuild_collection(collection,
+                                                                tree)
+    struct = tuple([collection])
```

Comment:
I like how Mulich adds a comment to denote what is the important part of a test, so do it above this line: add a comment `#  TARGET`